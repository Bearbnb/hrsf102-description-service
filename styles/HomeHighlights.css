.container {
  border: solid	#E0E0E0 1px;
  max-width: 600px;
  font-family: Circular,"Helvetica Neue",Helvetica,Arial,sans-serif;
  border-radius: 5px;
  box-shadow: 0 0 2px #C8C8C8;
  margin: 10px auto 10px auto;
  color: #484848;
}

.title {
  font-size: 12px;
  margin: 20px 0px 0px 15px;
}