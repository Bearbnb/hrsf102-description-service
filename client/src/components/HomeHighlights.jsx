import React from 'react';
import PropTypes from 'prop-types';

import HomeHighlight from './HomeHighlight.jsx';
import styles from '../../../styles/HomeHighlights.css';

const HomeHighlights = ({ homehighlights }) => (
  <div className={styles.container}>
    <div className={styles.title}>HOME HIGHLIGHTS</div>
    {homehighlights.map((highlight, i) => <HomeHighlight key={i} highlight={highlight} />)}
  </div>
);

HomeHighlights.propTypes = {
  homehighlights: PropTypes.arrayOf(PropTypes.object),
};

HomeHighlights.defaultProps = {
  homehighlights: [],
};
export default HomeHighlights;
