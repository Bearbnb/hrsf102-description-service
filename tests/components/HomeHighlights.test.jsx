import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import HomeHighlights from '../../client/src/components/HomeHighlights';

describe('<HomeHighlights />', () => {
  it('should render properly', () => {
    const wrapper = shallow(<HomeHighlights />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
