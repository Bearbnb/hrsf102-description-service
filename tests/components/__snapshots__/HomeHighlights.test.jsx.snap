// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeHighlights /> should render properly 1`] = `
<div
  className="container"
>
  <div
    className="title"
  >
    HOME HIGHLIGHTS
  </div>
</div>
`;
